framework/src/base/Assembly.C

```diff
@@ -1980,9 +1983,29 @@ Assembly::modifyWeightsDueToXFEM(const Elem * elem)
     mooseAssert(xfem_weight_multipliers.size() == _current_JxW.size(),
                 "Size of weight multipliers in xfem doesn't match number of quadrature points");
     for (unsigned i = 0; i < xfem_weight_multipliers.size(); i++)
-    {
       _current_JxW[i] = _current_JxW[i] * xfem_weight_multipliers[i];
-    }
+
     xfem_weight_multipliers.release();
   }
 }
+
+void
+Assembly::modifyFaceWeightsDueToXFEM(const Elem * elem, unsigned int side)
+{
+  mooseAssert(_xfem != NULL, "This function should not be called if xfem is inactive");
```

Comment:
This will trip only in debug, but is probably ok, because it is called by internally, correct?

Reply:
Yes, it is only called internally.